packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -1,5 +1,9 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IDeliveryStream } from '@aws-cdk/aws-kinesisfirehose-alpha';
```

Comment:
@paulhcsun I saw you working on this module. Can I use this here??

Reply:
Thanks, @paulhcsun! looks good from my view too. I'm just importing `IDeliveryStream` here.